WORKSPACE

```diff
@@ -30,80 +46,25 @@ http_archive(
     ],
 )
 
-# Needed as a transitive dependency of rules_webtesting below.
-http_archive(
-    name = "io_bazel_rules_go",
-    sha256 = "b7a62250a3a73277ade0ce306d22f122365b513f5402222403e507f2f997d421",
-    urls = [
-        # tag 0.16.3 resolves to commit 01e5a9f8483167962eddd167f7689408bdeb4e76 (2018-11-28 16:28:45 -0500)
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/0.16.3/rules_go-0.16.3.tar.gz",
-        "https://github.com/bazelbuild/rules_go/releases/download/0.16.3/rules_go-0.16.3.tar.gz",
-    ],
-)
-
-# Needed as a transitive dependency of rules_webtesting below.
-http_archive(
-    name = "bazel_gazelle",
-    sha256 = "6e875ab4b6bf64a38c352887760f21203ab054676d9c1b274963907e0768740d",
-    urls = [
-        # tag 0.15.0 resolves to commit c728ce9f663e2bff26361ba5978ec5c9e6816a3c (2018-10-13 00:06:11 +0200)
-        "https://mirror.bazel.build/github.com/bazelbuild/bazel-gazelle/releases/download/0.15.0/bazel-gazelle-0.15.0.tar.gz",
-        "https://github.com/bazelbuild/bazel-gazelle/releases/download/0.15.0/bazel-gazelle-0.15.0.tar.gz",
-    ],
-)
-
-http_archive(
-    name = "io_bazel_rules_webtesting",
-    sha256 = "89f041028627d801ba3b4ea1ef2211994392d46e25c1fc3501b95d51698e4a1e",
-    strip_prefix = "rules_webtesting-0.2.2",
-    urls = [
-        # tag 0.2.2 resolves to commit 596d07c1f38486486969302158b9019418a5409e (2018-12-04 09:20:24 -0800)
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_webtesting/archive/0.2.2.tar.gz",
-        "https://github.com/bazelbuild/rules_webtesting/archive/0.2.2.tar.gz",
-    ],
+load("@io_bazel_rules_closure//closure:defs.bzl", "closure_repositories")
+closure_repositories(
+    omit_com_google_protobuf = True,
+    omit_com_google_protobuf_js = True,
 )
 
 http_archive(
     name = "org_tensorflow",
-    sha256 = "a0d7eea4c4e4b526cdba624f706bfbcc87a7923bb09904dfab22589a6bd142f2",
-    strip_prefix = "tensorflow-6ef428bd6e83b0930266bf922eaa2f4a60e8328a",
+    sha256 = "1086f63c2c9fbea6873e137d08b5711a0493a5b699f258774da97f7672ba939a",
+    strip_prefix = "tensorflow-2243bd6ba9b36d43dbd5c0ede313853f187f5dce",
     urls = [
-        "https://mirror.bazel.build/github.com/tensorflow/tensorflow/archive/6ef428bd6e83b0930266bf922eaa2f4a60e8328a.tar.gz",  # 2018-12-06
-        "https://github.com/tensorflow/tensorflow/archive/6ef428bd6e83b0930266bf922eaa2f4a60e8328a.tar.gz",
+        "https://mirror.bazel.build/github.com/tensorflow/tensorflow/archive/2243bd6ba9b36d43dbd5c0ede313853f187f5dce.tar.gz",  # 2019-03-26
+        "https://github.com/tensorflow/tensorflow/archive/2243bd6ba9b36d43dbd5c0ede313853f187f5dce.tar.gz",
```

Comment:
why does this require an update?

Reply:
Two patches to TensorFlow Bazel files:

  - https://github.com/tensorflow/tensorflow/commit/494d2cc5cfd04c3c376f10b7d7c417e388b08282
  - https://github.com/tensorflow/tensorflow/commit/2243bd6ba9b36d43dbd5c0ede313853f187f5dce

Each of these fixes an `--incompatible_*` flag that otherwise hits us.